jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/roles/CaseRoleItemView.java

```diff
@@ -16,64 +16,218 @@
+                caseRoleAssignmentSummary.getModel().getUsers().forEach(user ->
+                        itemsList.add(new CaseRolesPresenter.CaseAssignmentItem() {
+                            @Override
+                            public String label() {
+                                return user;
+                            }
+
+                            @Override
+                            public void execute() {
+                                confirmPopup.show(translationService.format(REMOVE_ASSIGNMENT),
+                                        translationService.format(REMOVE),
+                                        translationService.format(REMOVE_USER_ASSIGNMENT_FROM_ROLE, user, caseRoleAssignmentSummary.getModel().getName()),
+                                        () -> presenter.removeUserFromRole(user, caseRoleAssignmentSummary.getModel().getName()));
+                            }
+                        }));
+                usersItemsLine.initWithItemsLine(maxWidth, name.getTextContent() + USERS_LINE_ID, itemsList);
+            }
+            if (caseRoleAssignmentSummary.getModel().getGroups().size() == 0) {
+                displayUnassignedGroup();
+            } else {
+                List<CaseRolesPresenter.CaseAssignmentItem> itemsList = new ArrayList<>();
+                caseRoleAssignmentSummary.getModel().getGroups().forEach(group ->
+                        itemsList.add((new CaseRolesPresenter.CaseAssignmentItem() {
+                            @Override
+                            public String label() {
+                                return group;
+                            }
+
+                            @Override
+                            public void execute() {
+                                confirmPopup.show(translationService.format(REMOVE_ASSIGNMENT),
+                                        translationService.format(REMOVE),
+                                        translationService.format(REMOVE_GROUP_ASSIGNMENT_FROM_ROLE, group, caseRoleAssignmentSummary.getModel().getName()),
+                                        () -> presenter.removeGroupFromRole(group, caseRoleAssignmentSummary.getModel().getName()));
+
+                            }
+                        })));
+                groupsItemsLine.initWithItemsLine(maxWidth, name.getTextContent() + GROUP_LINE_ID, itemsList);
+            }
+        }
     }
 
-    public void setIconType(final String iconTypeClass) {
-        addCSSClass(this.iconType, iconTypeClass);
+    public void setOwnerAssignment() {
+        usersItemsLine.initWithSingleItem(name.getTextContent(), caseRoleAssignmentSummary.getModel().getUsers().get(0));
+
+        addCSSClass(usersDiv, "kie-owner-role-height");
+        addCSSClass(groupsDiv, "hidden");
+        removeCSSClass(actions, "hidden");
+        addCSSClass(actionsItems, "hidden");
+        addCSSClass(actionsIcon, "kie-unassigned-color");
+    }
```

Comment:
Mouse cursor implies that the actions dropdown is clickable when hovering over it, although actions are disabled